lib/watir/attribute_helper.rb

```diff
@@ -38,6 +38,10 @@ def attribute_list
     #  @return [$1] value of $3 property
     #
     def attribute(type, method, attr)
+      if %i(thead tfoot).include? method
```

Comment:
Should I move these there as well? https://github.com/watir/watir/blob/master/lib/watir/attribute_helper.rb#L41